docs/manage-table-partition-using-gravitino.md

```diff
@@ -0,0 +1,370 @@
+            },
+            Maps.newHashMap());
+```
+
+:::note
+The values are in the same order as the field names.
+
+When adding an identity partition to a Hive partitioned table, the specified partition name is disregarded. This is because Hive generates the partition name based on field names and values.
+:::
+
+</TabItem>
+<TabItem value="range" label="Range">
+
+```java
+Partition partition =
+        Partitions.range(
+            "p20200321",
+            Literals.dateLiteral(LocalDate.parse("2020-03-21")),
+            Literals.NULL,
+            Maps.newHashMap());
+```
+
+</TabItem>
+
+<TabItem value="list" label="List">
+
+```java
+Partition partition =
+        Partitions.list(
+            "p202204_California",
+            new Literal[][] {
+              {
+                Literals.dateLiteral(LocalDate.parse("2022-04-01")),
+                Literals.stringLiteral("Los Angeles")
+              },
+              {
+                Literals.dateLiteral(LocalDate.parse("2022-04-01")),
+                Literals.stringLiteral("San Francisco")
+              }
+            },
+            Maps.newHashMap());
+```
+
+:::note
+Each list in the lists must have the same length. The values in each list must correspond to the field definitions in the list [partitioning](./table-partitioning-bucketing-sort-order-indexes.md#table-partitioning).
+:::
+
+</TabItem>
+</Tabs>
+
+You can add a partition to a partitioned table by sending a `POST` request to the `/api/metalakes/{metalake_name}/catalogs/{catalog_name}/schemas/{schema_name}/tables/{partitioned_table_name}/partitions` endpoint or by using the Gravitino Java client.
+The following is an example of adding a range partition to a partitioned table:
+
+<Tabs>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" -d '{
+  "partitions": [
+    {
+      "type": "range",
+      "name": "p20200321",
+      "upper": {
+        "type": "literal",
+        "dataType": "date",
+        "value": "2020-03-21"
+      },
+      "lower": {
+        "type": "literal",
+        "dataType": "null",
+        "value": "null"
+      }
+    }
+  ]
+}' http://localhost:8090/api/metalakes/metalake/catalogs/catalog/schemas/schema/tables/table/partitions
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+GravitinoClient gravitinoClient = GravitinoClient
+    .builder("http://127.0.0.1:8090")
+    .build();
+
+// Assume that you have a partitioned table named "metalake.catalog.schema.table".
+Partition addedPartition = 
+    gravitinoClient
+        .loadMetalake(NameIdentifier.of("metalake"))
+        .loadCatalog(NameIdentifier.of("metalake", "catalog"))
+        .asTableCatalog()
+        .loadTable(NameIdentifier.of("metalake", "catalog", "schema", "table"))
+        .supportPartitions()
+        .addPartition(
```

Comment:
It is better to use Identity partition as an example. Currently, we don't support range and list partition.

Reply:
fixed